src/store/reducers/manipulateTodoState.js

```diff
@@ -0,0 +1,155 @@
+import * as actionTypes from '../actions/actionTypes';
```

Comment:
I think the file name could be `store/reducers/todos`.

Reply:
I will change it !